requirements.txt

```diff
@@ -9,4 +9,6 @@ beautifulsoup4>=4.11.1
 peewee>=3.16.2
 requests_cache>=1.0
 requests_ratelimiter>=0.3.1
-scipy>=1.6.3
\ No newline at end of file
+scipy>=1.6.3
+protobuf>=5.29.2
+websockets>=14.1
```

Comment:
Are the almost-latest versions really necessary? Not impossible that users have packages that require older.